config/sync/migrate_plus.migration.d7_file_media_image.yml

```diff
@@ -19,26 +19,33 @@ source:
   constants:
     bundle: image
 process:
-  mid: fid
   bundle: constants/bundle
   langcode:
     plugin: default_value
     source: language
     default_value: und
-  name: field_file_image_title_text/0/value
+  name: filename
```

Comment:
Ah, need to put in something to coalesce these two... image title over filename, where it's given.